src/components/LeafletMap.js

```diff
@@ -200,7 +201,9 @@ export class LeafletMap {
       });
     }
 
-    this.dataLayers.set(layerConfig.name, {
+    // Create layer group with localised name
+    const localizedName = i18next.t(layerConfig.nameI18n) || layerConfig.name;
```

Comment:
Good point, but I think yes since they all depend on async data loading which definitely gets added to the queue after i18next inits. It's not explicit, which I don't like, but the map layers loading also depend on the network, which basically assures it will happen after the i18next inits... However I don't really like solving it this way.

Maybe we should put all of the class initialisations in `index.js` in to a `then()` after the i18n is initialised. I can even do that by extending the `i18nInit` to return a promise?

Reply:
Having the `init` function return a Promise and wrapping the class initializations in a `.then()` tacked on to it seems like the most sensible thing to do. The `getData()` call could also be wrapped in a `.then()` chained on as well. To clean things up you could also use `async / await` in the `index.js` module to wait for the `i18nInit` to resolve as well as `catch` any errors that might occur.

Reply:
I noticed that the `i18n.init()` call returns the `t` function on success, so we could also pass it to class methods there as well, if desired.

Reply:
I was going to push these changes here @djbusstop but something got messed up on my end with the branch tracking. Take a look at the branch [clhenrick/iss16-i18n](https://github.com/antievictionmappingproject/covid-19-map/tree/clhenrick/iss16-i18n) and commits 2892a0cc7117f8a8b82b80a01868ce86e2bb182b and 3c0dfcfb423d0b92fb335d5e5910a411b6bb8a75
to see how I did it if you'd like.

Reply:
I don't think we should pass the t around, since i18n is designed to be used the way I have used it. I will wait however for the i18n to load before anything else.